package.json

```diff
@@ -4,7 +4,7 @@
   "description": "bulid an expressjs server",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "nodemon server.js",
```

Comment:
You need to add `nodemon` to the `package.json`

Reply:
I did https://github.com/ioulungTsai/expressJS-server/blob/174f56ad2bfd4783d6e7e7b157028b49ca3c0bcb/package.json#L25